app/src/main/java/com/example/joshuamugisha/github/adapter/GithubAdapter.java

```diff
@@ -48,7 +48,7 @@ public void onBindViewHolder(GithubUserViewHolder holder, int position) {
         holder.name.setText(githubUser.getLogin());
         Glide.with(mContext)
                 .load(githubUser.getAvatarUrl())
-                .apply(RequestOptions.circleCropTransform())
+//                .apply(RequestOptions.circleCropTransform())
```

Comment:
Kindly remove all the code that you find not necessary before committing.

Reply:
@Estaer I have worked on this